### DISCUSSION TITLE
questions code

### DISCUSSION THREAD

**Discussion by Marine1719** [BIO6032/2022-SEIR-reproduction#39] [General]
Salut tout le monde, j'essaye de refaire le modèle en changeant les valeurs mais je beug sur cette ligne, qql peut m'expliquer pourquoi on a écrit la fonction cumulative comme telle ? Pourquoi n'avait-on pas pris la somme de ce qui donne les cas positifs (donc les A + I + Q ) ? Comment avait on définit cette ligne ?? 

C = cumsum((_Ss + _s*_ft*_Tr) * (A/_Tr))

Merci :)

**Comment by salomebsk** [comment]
Salut!
On l'a basé sur cette équation en bas à gauche de la page 3 de l'article:
![image](https://user-images.githubusercontent.com/82335200/162214424-06a84601-a54e-4c3c-bc94-f2eb345309c5.png)

:)

**Comment by Marine1719** [comment]
Ah oui juste j'avais oublié je pensais qu'on était reparti d'une autre équation je sais pas pourquoi..  désolée Merci!

**Reply by salomebsk** [reply]
avec plaisir! :)
